web/libs/core/src/lib/utils/safe-fetch.ts

```diff
@@ -0,0 +1,56 @@
+type FetchParams = Parameters<typeof fetch>;
+
+// We're catching certain types of errors that
+// are not supposed to be user-facing
+const IGNORED_ERRORS = new RegExp(
+  [
+    "abort", // Whenever the abort controller kicked in
+    "failed to fetch", // Network's offline, bad URL, CORS, etc. in Chrome
+    "networkerror", // Same as the above but Firefox,
+  ].join("|"),
+  "i",
+);
+
+/**
+ * Determines if an error should be silently ignored based on predefined patterns.
+ *
+ * This function checks if the error message contains any of the patterns defined in
+ * IGNORED_ERRORS (like "abort", "failed to fetch", "networkerror"). These are typically
+ * network-related errors that don't require user notification.
+ *
+ * @param error - The error object to check
+ * @returns true if the error should be ignored, false otherwise
+ */
+export function isErrorIgnored(error: unknown) {
+  if (!(error instanceof Error)) return false;
+  // we don't want the user to see some of the errors
+  // so we fail silently
+  if (error.message.match(IGNORED_ERRORS) !== null) {
+    return true;
+  }
+}
```

Comment:
for consistent results
```suggestion
  return false;
}
```